Tools/scripts/summarize_stats.py

```diff
@@ -120,6 +123,26 @@ def load_raw_data(input: Path) -> RawData:
 def save_raw_data(data: RawData, json_output: TextIO):
     json.dump(data, json_output)
 
+@functools.cache
+def _get_uop_flags_from_file(
+    flag_names: tuple[str] = None,
+    filepath: str | Path = "Include/internal/pycore_uop_metadata.h",
+) -> dict[str, list[str]]:
+    flags = {}
+    with open(SOURCE_DIR / filepath) as spec_src:
+        for line in spec_src:
+            if line.startswith("    [_"):
```

Comment:
If the indentation in that file changes, we might silently not find anything in the file.  Maybe use a regular expression to match "\s*[_" instead.

Reply:
Makes sense, updated.